MODERATION-GUIDELINES.md

```diff
@@ -47,3 +47,19 @@ The job of moderators is to enforce the community's desired norms of discussion,
 ## Feedback
 
 No system of moderation can be perfect or meet everyone's expectations. Too much discussion about moderation can also be a distraction and not a good use of contributors limited time. So discussion about moderation policies or decision should generally be avoided within Bitcoin Core's GitHub software projects, but discussion about moderation, and other topics that could make contributing to Bitcoin Core more productive and pleasant is welcome in the [bitcoin-core/meta](https://github.com/bitcoin-core/meta) GitHub project about project management issues.
+
+## Oversight
+
+Moderators are not perfect and will make mistakes.
+
+If moderation decisions are causing repeated problems, please request a review of moderation decisions by opening a public issue in https://github.com/bitcoin-core/meta/issues/new or email moderation@bitcoincore.org privately, so patterns of bad decisions can be discussed and corrected. Please include as much detail as you can provide about the specific moderation decisions which were unfair or counterproductive. If you think it is relevant, you can also include background information about things like past disagreements or conflicts of interest. Focus of the review will only be to improve moderation decisions and quality of discussion so please only include background information that is likely to be relevant to making better moderation decisions.
```

Comment:
> moderation@bitcoincore.org

I am confirming with the project maintainers that this email address will work. If there is an emergency, for now, `security@bitcoincore.org` may be used.

Reply:
It’s good to have distinct and (ideally public ) group of folks behind both endpoints.

I’ve already been in the situation years ago in the past of bugs reported to the list, and an apparent conflict of interest with other regular and still active contributors involved in security issues handling — and I won’t say more as it’s not my role to comment what people are doing with their private life. More it would be lack of decency.

Reply:
The email address is up and running, forwards to all current code maintainers